liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -106,8 +106,8 @@
     /**
      * Liberty output directory (<tT>WLP_OUTPUT_DIR</tt>).
      */
-    @Parameter(property = "outputDirectory")
-    protected File outputDirectory = null;
+    @Parameter(property = "outputDirectory", defaultValue = "${project.build.directory}/liberty-alt-output-dir")
```

Comment:
Will this have any undesired side effects? Previously, the outputDirectory was only set to this value for the install-feature and package goals in the parent pom. An alternative would be to set outputDirectory to this value in the constructor of those mojos. If having this default for all goals works just fine, then leave it. But you should also update the doc for common parameters to say this is the default value. And if you leave it here, you should undo your changes to the two mojos that you updated the copyright years for.